conformance-test/libraryMethods.ts

```diff
@@ -680,8 +680,8 @@ export async function deleteHMAC(options: ConformanceTestOptions) {
   const metadata = {
     state: 'INACTIVE',
   };
-  options.hmacKey!.setMetadata(metadata);
-  await options.hmacKey!.delete();
+  await hmacKey.setMetadata(metadata);
```

Comment:
does this not run into the problem where the instructions get used here?

Reply:
It does not.  I spoke with Cathy and verified in testing that only the delete call will utilize instruction count.

Reply:
oh right because the instructions are on the method specified. this makes sense.